src/alignment/__init__.py

```diff
@@ -1 +1,5 @@
 __version__ = "0.2.0.dev0"
```

Comment:
don't forget to change this.

Reply:
yeah i'll merge and then make a release for the "lib"